recipe/meta.yaml

```diff
@@ -35,13 +37,19 @@ requirements:
 test:
   imports:
     - zeep
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
   home: http://www.python-zeep.org
   license: MIT
   license_family: MIT
   license_file: LICENSE
   summary: A fast and modern Python SOAP client
+  doc_url: http://docs.python-zeep.org/
```

Comment:
We might want to remove the duplicate `doc_urls` and `dev_urls`

https://github.com/AnacondaRecipes/zeep-feedstock/blob/dea42e77e27a83237f65e5bb3e92f2dfbd95d3b9/recipe/meta.yaml#L51&L66

Reply:
Strange, I must of added those thinking there we not any. I went ahead and removed them